src/manifest.c

```diff
@@ -582,6 +582,16 @@ void subtract_manifests(struct manifest *m1, struct manifest *m2)
 			list1 = g_list_next(list1);
 			list2 = g_list_next(list2);
 
+			/* When both files are deleted, skip subtraction, since
```

Comment:
The same last_change case is more complex to think about. I'll revisit tomorrow.